### ISSUE TITLE
Verification set accuracy

### ISSUE THREAD

**Issue by Mr-watermelon** [brijeshiitg/Pytorch-Implementation-of-YeNet-Deep-Learning-Hierarchical-Representations-for-Image-Steganalysis-#1]
The accuracy of the training set can reach more than 90, but the verification set can only reach about 77. Is this normal?I see that's not the result in the paper?

**Comment by fennudehaogua** [comment]
I have the same experimental results， @brijeshiitg Can you provide the trained model like SRNet？

**Comment by huyppppppp** [comment]
> The accuracy of the training set can reach more than 90, but the verification set can only reach about 77. Is this normal?I see that's not the result in the paper?

hello, May I ask where the data set came from?

**Comment by huyppppppp** [comment]
> I have the same experimental results， @brijeshiitg Can you provide the trained model like SRNet？

Hello, May I ask where the data set come from?

**Comment by brijeshiitg** [comment]
Trained model is not available for YeNet. You can train it from scratch.
